prebuilt_forms/dynamic_elasticsearch.php

```diff
@@ -252,16 +252,13 @@ protected static function get_control_permissionFilters($auth, $args, $tabalias,
   }
 
   /**
-   * Output a selector for a general record access contexts based on permission filters and group permissions etc
```

Comment:
This looks a bit messy because there was some redundant code 'recordContext' stuff that I overlooked from previous commit. So ignore the deleted recordContext stuff.